components/search/SearchSuggestion.vue

```diff
@@ -170,7 +170,7 @@
         </template>
       </NeoTabItem>
     </NeoTabs>
-    <div v-if="!name" class="search-history">
+    <div v-if="!name" class="search-history pt-5 mt-0">
```

Comment:
```suggestion
    <div v-if="!name" class="search-history pt-5">
```

Reply:
this `mt-0` is useful

Reply:
looks like it dose not work as you have deleted the margin from `search-history`

Reply:
aya yes it is